conans/client/command.py

```diff
@@ -475,8 +475,7 @@ def info(self, *args):
                             "specify both install-folder and any setting/option "
                             "it will raise an error.")
         parser.add_argument("-j", "--json", nargs='?', const="1", type=str,
-                            help='Only with --build-order option, return the information in a json.'
-                                 ' e.g --json=/path/to/filename.json or --json to output the json')
+                            help='Path to a json file where the information will be written')
```

Comment:
I've intentionally removed the mention to `--json` (without value) will output to console, I think it is something we don't want to promote. Shall I revert it?

Reply:
why not promote? isn't such text interface used as a base for unix command-line pipelining, e.g. `conan --json | some_tool | some_other_tool`?

Reply:
We cannot make this commitment right now, there are some commands that are outputting a lot of information to the console and then generating the json data (like the `info` one with the profiles information), so you cannot pipe the output to the next tool.

It would be a nice-to-have, it will require to silent all the output and then just print the JSON data. Something to do, not so hard, but not yet.

More issues related to output: #4225 (what we are talking here), and two more issues also worth reading #4215, #4067

Reply:
I am not sure we can just remove this despite the fact that I don't like it as it is either. We will need to discuss it

Reply:
I'm talking about removing it just from the `--help` output.

Reply:
ok, it is fine for me